[2022-09-13 20:31:23,069] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_dag.notify manual__2022-09-13T20:31:21.377172+00:00 [queued]>
[2022-09-13 20:31:23,074] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_dag.notify manual__2022-09-13T20:31:21.377172+00:00 [queued]>
[2022-09-13 20:31:23,075] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 20:31:23,076] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-09-13 20:31:23,077] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 20:31:23,086] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): notify> on 2022-09-13 20:31:21.377172+00:00
[2022-09-13 20:31:23,090] {standard_task_runner.py:52} INFO - Started process 963 to run task
[2022-09-13 20:31:23,093] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'notify', 'manual__2022-09-13T20:31:21.377172+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/pre_parser.py', '--cfg-path', '/tmp/tmpt_hm9s3a', '--error-file', '/tmp/tmpstx3atqg']
[2022-09-13 20:31:23,094] {standard_task_runner.py:80} INFO - Job 28: Subtask notify
[2022-09-13 20:31:23,139] {task_command.py:369} INFO - Running <TaskInstance: test_dag.notify manual__2022-09-13T20:31:21.377172+00:00 [running]> on host 5d92941617fc
[2022-09-13 20:31:23,226] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=notify
AIRFLOW_CTX_EXECUTION_DATE=2022-09-13T20:31:21.377172+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-13T20:31:21.377172+00:00
[2022-09-13 20:31:23,231] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-13 20:31:23,232] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "Hello!"']
[2022-09-13 20:31:23,244] {subprocess.py:85} INFO - Output:
[2022-09-13 20:31:23,250] {subprocess.py:92} INFO - Hello!
[2022-09-13 20:31:23,251] {subprocess.py:96} INFO - Command exited with return code 0
[2022-09-13 20:31:23,276] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=notify, execution_date=20220913T203121, start_date=20220913T203123, end_date=20220913T203123
[2022-09-13 20:31:23,305] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-13 20:31:23,326] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
